build.gradle

```diff
@@ -66,7 +66,9 @@ dependencies {
 }
 
 shadowJar {
-    archiveFileName = 'addressbook.jar'
+    archiveBaseName = 'BandBook'
+    archiveVersion = 'v1.2'
+    archiveClassifier = ''
```

Comment:
Good to indicate the version number so that we can easily generate the JAR file with the proper naming! 👍